Assets/MixedRealityToolkit.Tools/OptimizeWindow/MixedRealityOptimizeWindow.cs

```diff
@@ -105,10 +105,11 @@ private void OnGUI()
             MixedRealityInspectorUtility.RenderMixedRealityToolkitLogo();
 
             // Render Title
-            EditorGUILayout.BeginHorizontal();
+            using (new EditorGUILayout.HorizontalScope())
```

Comment:
Is there a benefit to this change, or just more clear scoping?

Reply:
I have started switching to it because I like the clear scope. It exceedingly more difficult in complicated inspectors otherwise

Reply:
I agree with Troy. This pattern is cleaner and more robust (you cannot forget to call EndHorizontal)